### ISSUE TITLE
Platform Integration March 2017

### ISSUE THREAD

**Issue by zelima** [cybergreen-net/pm#94]
### Summary
Show a thematic integration of info site and stats site (fonts, style, etc.) 

[Google doc](https://docs.google.com/document/d/11mx102LteFY6dxyQchOTM9sd_9XVTOykIdbrfRB16f4/edit)

### Tasks
- [x] Provide screen shots in Google doc

**Comment by rufuspollock** [comment]
@zelima you should own this - not @shiweiart Please can you action this -- and ping @shiweiart if you need help.

**Comment by zelima** [comment]
FIXED. @ataslim this include two simple screenshots one from www and one from stats site. Please reopen if we need more.

**Comment by ataslim** [comment]
I think there are issues with uniformity of CSS across main site and stats site. Look at the headers on the metrics page. Shouldn't they be green if we are using the theme from the info site?

If we can make it uniform across both sites and cite what fonts/styles we use in the report, i think we'll be all set.

**Comment by rufuspollock** [comment]
@ataslim we can check css stuff and I think you are right that at h3 level we may not be matching styles (@zelima can you check this).

we've done one pass already and I should note that even with similar css there may be some slightly different look as it is hard to align absolutely perfectly because we are using two different html/css frameworks -- bootstrap and bourbon.

@zelima note that metrics page is going to move off this site and consolidate into the glossary we talked about which should be on the main site I think.

**Comment by zelima** [comment]
@ataslim @rufuspollock this PR aligns styling with main website. Note that we use different green for h1 tags but as we discussed on Thursday call we need more "strict" colors for statistics website. 

We use h3 only on metrics page and yes it's slightly different, but as it is going to move off this site, it will not remain as issue any more...

I updated doc with new screen-shots. 

Will leave this open for review for a while
